opcua/server/server.py

```diff
@@ -99,6 +99,12 @@ def __init__(self, shelffile=None, iserver=None):
         sa_node = self.get_node(ua.NodeId(ua.ObjectIds.Server_ServerArray))
         sa_node.set_value([self._application_uri])
 
+        self.security_endpoints = ["None", "Basic128Rsa15_Sign",
+                                    "Basic128Rsa15_SignAndEncrypt", 
+                                    "Basic256_Sign", "Basic256_SignAndEncrypt"]
+        self.policyIDs = ["Anonymous", "Basic256", "Basic128", "Username"]
```

Comment:
didn't you plan to make this variables private and add two methods? something like set_security_modes and set_user_identification_modes? I think this will improve discoverability and doc